app/web/features/mod/ModPage.tsx

```diff
@@ -0,0 +1,38 @@
+import { Typography } from "@mui/material";
+import Button from "components/Button";
+import HtmlMeta from "components/HtmlMeta";
+import PageTitle from "components/PageTitle";
+import UsersList from "components/UsersList";
+import { useTranslation } from "i18n";
+import { GLOBAL, MOD, PROFILE } from "i18n/namespaces";
+
+import { useNewUsers } from "./hooks";
+
+export default function ModPage() {
+  const { t } = useTranslation([GLOBAL, MOD, PROFILE]);
+
+  const { userIds, error, hasNextPage, isFetchingNextPage, fetchNextPage } =
+    useNewUsers();
+
+  return (
+    <>
+      <HtmlMeta title={t("mod:title")} />
+      <PageTitle>{t("mod:title")}</PageTitle>
+      <h1>{t("mod:list_users.heading")}</h1>
+      <UsersList
+        error={error}
+        userIds={userIds}
+        emptyListChildren={
+          <Typography variant="body1">
+            {t("mod:list_users.none_found")}
+          </Typography>
+        }
```

Comment:
Rather than pass a node, I'd just remove this prop (make it conditional) and check if it's undefined within `UsersList` then show the Typography no users found there. That keeps that component responsible for its own logic.

Reply:
There is a minor difference in behavior though: with `emptyListChildren`, it will show an empty list if the users are blocked out for some reason. E.g. in your blocking PR, if `userIds` consisted of exactly one blocked user, then this would display the `emptyListChildren` node (because it runs it through `useLiteUsers`).